docs/release_notes.md

```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## Version 1.0.4
+
+### Bug Fixes
+
+- Updated thirdparty libraries to fix a bug in the licensing system that was preventing the activation of the product in some cases. *AdonisFX-1185*
+
```

Comment:
I would mention the limelm error number so if at any point any user has this problem can easily confirm that the new version would fix the problem.

Reply:
Very good suggestion. Applied!